manual/manual/cmds/intf-c.etex

```diff
@@ -1607,6 +1607,42 @@ by calling "Printexc.record_backtrace true" in the initialization of
 one of the OCaml modules.  This can also be achieved from the C side
 by calling "caml_record_backtrace(Val_int(1));" in the OCaml-C glue code.
 
+\paragraph{Unloading the runtime.}
+
+In case the shared library produced with "-output-obj" is to be loaded and
+unloaded repeatedly by a single process, care must be taken to unload the
+OCaml runtime explicitly, in order to avoid various system resource leaks.
+
+Since 4.05, "caml_shutdown" function can be used to shut the runtime down
+gracefully, which equals the following:
+\begin{itemize}
+\item Running the finalizers that were registered with "Pervasives.at_exit".
```

Comment:
Use `functions` instead of `finalizers`. We already have two kinds of finali[sz]ers in OCaml, and none of them have anything to do with `at_exit`.